tskv/src/memcache.rs

```diff
@@ -80,6 +80,46 @@ impl MemEntry {
     }
 }
 
+pub struct MemRaw<'a> {
+    pub seq: u64,
+    pub ts: Timestamp,
+    pub field_id: FieldId,
+    pub field_type: ValueType,
+    pub val: &'a [u8],
+}
+
+impl MemRaw<'_> {
+    pub fn insert(
```

Comment:
Does this method means 'insert `this` into specified `cache`, then update specified `cache_size`?'